index.html

```diff
@@ -257,19 +301,19 @@ <h5>Languages</h5>
                   <li>
                     <div class="Q">
                       <img src="images/java.png" alt="" />
-                      <p>JavaScript</p>
+                      <p class="item-1">JavaScript</p>
                     </div>
                   </li>
                   <li>
                     <div class="Q">
                       <img src="images/html.png" alt="" />
-                      <p>HTML</p>
+                      <p class="item-1">HTML</p>
                     </div>
                   </li>
                   <li>
                     <div class="Q">
                       <img src="images/css.png" alt="" />
-                      <p>CSS</p>
+                      <p class="item-1">CSS</p>
                     </div>
```

Comment:
- [ ] As requested by the previous reviewer please implement the changes. please make sure that it matches the design like spacing.
### Your design
![hissssssss](https://user-images.githubusercontent.com/81727988/194684337-18811651-acde-4aad-91f7-e2ff7ff9f03f.png)
### Figma design
'
![fimgaaaaaaaa](https://user-images.githubusercontent.com/81727988/194684346-be16dec5-ee1f-421d-bdf5-0738b01f97a6.png)

Reply:
As the previous reviewer suggested, this section of your page does not exactly match the design in the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
Please pay attention to the following:
-- The spacing between the `about myself section` and the `languages` section doesn't match what is in the design
--  The languages section doesn't match the Figma design
-- The top-border of the section is missing in your design.
<img width="947" alt="languages" src="https://user-images.githubusercontent.com/73607512/194710045-0388ab16-be6c-4dbb-a403-50742273e678.PNG">

Figma template
<img width="539" alt="languages and about figma" src="https://user-images.githubusercontent.com/73607512/194710112-03f0107a-70e1-4495-bfe4-35248e05c27e.PNG">